client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -281,9 +283,14 @@ def test_form_update_body(self):
             Memory("500Mi"),
         ]
 
-        parameter_json = endpoint._form_update_body(resources, DirectUpdateStrategy(), 0)
-        assert parameter_json == {'build_id': 0, 'resources': {'cpu_millis': 500, 'memory': '500Mi'}, 'strategy': 'rollout'}
-        
+        env_vars = {'CUDA_VISIBLE_DEVICES': "1.2", "VERTA_HOST": "app.verta.ai"}
```

Comment:
Not important, but this environment variable intentionally uses a comma!
```suggestion
        env_vars = {'CUDA_VISIBLE_DEVICES': "1,2", "VERTA_HOST": "app.verta.ai"}
```